Posit Infrastructure/Posit Environment Quick Start Guide.md

```diff
@@ -0,0 +1,249 @@
+# Posit Environment Quick Start Guide
+
+## 1. Overview
+
+### Purpose
+This Quick Start Guide provides essential information and step-by-step instructions to get you started with the Posit Environment on day one.  It is not a comprehensive guide; further documentation and guidance will continue to be added and updated on the [PHS Data Science Knowledge Base](https://public-health-scotland.github.io/knowledge-base/).
+
+### Audience
+This guide assumes some previous experience with the R programming language, the RStudio Integrated Development Environment (IDE), and perhaps an earlier version of Posit Workbench.
+
+### Scope
+This guide covers:
+- A high-level overview of the Posit Environment's specifications and features.
+- Step-by-step instructions to get you started with logging in and setting up your environment.
+- Details on how to get help.
+- Links to other useful information and resources.
+
+---
+
+## 2. Environment Specifications
+
+### Operating System and Application Specifications
+
+- **Operating System**
+  - Ubuntu 22.04.5 LTS Jammy Jellyfish
+- **Applications and Versions**:
+  - **Posit Workbench**: 2025.09.1+401.pro2, “Cucumberleaf Sunflower” (59cf440a).
+  - **Posit Package Manager**: v2025.09.0-7
+  - **Posit Connect**: _Coming soon..._
+- **R and Python Versions**:
+  - **R**: Default version is v4.5.1, with additional support for v4.4.2 
+  - **Python**: Default version is 3.13.8, with additional support for versions 3.12.11 and 3.10.12
+- **Supported IDEs**:
+  - RStudio Pro 2025.09.1 Build 401.pro2 with support for Quarto 1.8.25
+  - Jupyter Notebook 7.2.3
+  - JupyterLab 4.2.7
+  - Visual Studio Code 1.102.0
+  - Positron Pro 2025.08.1 build 11 (for evaluation purposes only)
```

Comment:
Might be useful to turn some of these versions into links (where there's a useful link to provide context).

- https://www.jumpingrivers.com/blog/whats-new-r45/
- https://docs.posit.co/ide/news/#rstudio-2025.09.1
- https://positron.posit.co/features.html